tests/tests/test_filetransfer_cli.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2021 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import os
+import subprocess
+import time
+
+from tempfile import NamedTemporaryFile
+
+from ..common_setup import standard_setup_one_client
+from ..MenderAPI import authentication, devauth, get_container_manager, logger
+from .common_connect import wait_for_connect
+from .common import md5sum
+from .mendertesting import MenderTesting
+
+
+class TestFileTransferCLI(MenderTesting):
+    """Tests the file transfer functionality"""
+
+    def test_filetransfer_cli(self, standard_setup_one_client):
+        # accept the device
+        devauth.accept_devices(1)
```

Comment:
I've seen this already in other tests that you @tranchitella write. Is there a good reason for not using `standard_setup_one_client_bootstrapped` instead? The framework will then accept the device and your test only starts after/if is accepted.

Reply:
uhm, I believe it's a copy-and-paste from other tests. I can try rewriting it using the `standard_setup_one_client_bootstrapped` indeed, let me try it.

Reply:
Done: https://github.com/mendersoftware/integration/pull/1274/commits/e6d4bff1d6348de853d4fc21b40cfc006af57c62